### DISCUSSION TITLE
Add support of Ultimate shop

### DISCUSSION THREAD

**Discussion by YelaWolf58** [BG-Software-LLC/WildChests#232] [Pending Feature Requests]
Please can you add this plugin for wildchest

**Comment by OmerBenGera** [comment]
this?
https://www.spigotmc.org/resources/ultimateshop-menu-dynamic-price-limits-custom-condition-sell-all-and-more-1-14-1-20-6.110601/

**Reply by YelaWolf58** [reply]
Yes

**Comment by OmerBenGera** [comment]
Their API and their codebase in general is just awful
I am not gonna add support for it as of now, sorry.
